public/_data.json

```diff
@@ -4,45 +4,45 @@
       "datetime": "Thursday August 31st, 2017 7pm",
       "date": "8/31/2017 19:00:00",
       "host": {
-        "address": "2401 Colorado Ave, Santa Monica, CA 90404",
-        "name": "Edmunds",
-        "misc": "Parking validation provided in Colorado Center (company parking lot).",
-        "url": "https://www.edmunds.com/"
+        "address": "2110 Broadway, Santa Monica, CA 90404",
+        "name": "TigerText",
+        "misc": "Nearby street parking and some spaces available in their lot",
+        "url": "https://www.tigertext.com/"
```

Comment:
should this be the link to their site or the jobs board that we link to in email/eventbrite?